WordPress/gutenberg
# [Widgets screen] Fix categorization not showing in the global inserter

## Description

Fix #26159 and #26329.

Removed the now outdated check of `hasChildItems`.

## How has this been tested?



1. Go to the widgets screen.
2. Open the global inserter.
3. There should be categorization of blocks.

## Screenshots 
![image](https://user-images.githubusercontent.com/7753001/103865091-9c5a2200-50fe-11eb-9d8c-b249f2044acf.png)

## Types of changes




Bug fix

## Checklist:
